dll/win32/mswsock/CMakeLists.txt

```diff
@@ -16,6 +16,6 @@ add_library(mswsock MODULE
     ${CMAKE_CURRENT_BINARY_DIR}/mswsock.def)
 
 set_module_type(mswsock win32dll UNICODE)
-add_importlibs(mswsock ws2_32 advapi32 dnsapi msvcrt kernel32 ntdll)
+add_importlibs(mswsock ws2_32 advapi32 dnsapi msvcrt kernel32 user32)
```

Comment:
On Windows: there is a ntdll dependence (it is there because the NT IO functions are used), and user32 is delay-loaded, only when necessary.